geoips/geoips_utils.py

```diff
@@ -78,78 +76,31 @@ def load_all_yaml_plugins():
 
     reg_plug_path = resources.files("geoips") / "registered_plugins.yaml"
     if not os.path.exists(reg_plug_path):
-        from . import create_plugin_registry
-
-        create_plugin_registry.main()
+        raise PluginRegistryError(
+            f"Plugin registry {reg_plug_path} did not exist, "
+            "please run 'create_plugin_registry'"
+        )
```

Comment:
Note this is where we would end up with circular imports if we tried to run create_plugin_registry during runtime (since we now need geoips.interfaces.products utilities in order to find the correct products to populate the registered_plugins.yaml files).